game-rules.tex

```diff
@@ -4,16 +4,25 @@ \section{Game Rules}
 \begin{enumerate}
   \item The game, called \emph{\gamename}, is played in the arena defined in
         Specification~\ref{spec:arena}. The objective is to collect tokens and
-        deposit them in the teams scoring zone.
-  \item The arena contains 16 tokens.
-  \item There are four scoring zones in the arena, arranged in a $2\times2$
-        grid, with each robot being allocated a single scoring zone. The centre
-        of the arena (approximately a quarter of each scoring zone) is raised.
+        deposit them in scoring zones.
+  \item The arena contains 24 tokens.
+  \item There are nine scoring zones in the arena, arranged in a $3\times3$
+        grid. The central scoring zone is raised.
+  \item A scoring zone is considered to be `controlled' by a team if and only if:
+    \begin{enumerate}
+      \item the team has tokens in the zone.
+      \item no other team has a greater than, or equal to number of tokens in the zone.
```

Comment:
I would suggest that scoring zones with equal tokens of different teams should be 'controlled' by both teams. Teams will probably not get that many tokens positioned strategically anyway, so it feels like this could lead to lots of low/no scoring games

Reply:
We agreed last night that both teams would control the zone in this circumstance.